clear-site-data/storage.https.html

```diff
@@ -45,6 +69,18 @@
                 }, test_name);
               });
 
+              promise_test(function() {
+                const serviceWorkerResponseOk = TestUtils.getServiceWorkerResponseOk();
+
+                // Tell the iframe to make a request for the URL that the service worker
+                // responds to
+                serviceWorkerTestPageIFrame.contentWindow.postMessage(null, "*");
+
+                return serviceWorkerResponseOk.then(function(responseIsOk) {
+                  assert_false(responseIsOk, "Response should not be 200");
+                });
+              }, "Service worker no longer responds to requests");
```

Comment:
It would also be useful to check that the iframe's `navigator.serviceWorker.controller` is null and `controllerchange` fired in the iframe.

Reply:
True dat. Added these assertions.